index.bs

```diff
@@ -3997,6 +4005,7 @@ partial interface MLGraphBuilder {
             1. Let |desc2| be a new {{MLOperandDescriptor}}.
             1. Set |desc2|.{{MLOperandDescriptor/dataType}} to |input|'s [=MLOperand/dataType=].
             1. Set |desc2|.{{MLOperandDescriptor/dimensions}} to the [=/list=] « |steps|, |numDirections|, |batchSize|, |hiddenSize| ».
+        1. If running the [=MLActivation/validation steps=] of any [=list/item=] in |options|.{{MLLstmOptions/activations}} with |desc| returns false, then [=exception/throw=] a {{TypeError}}.
```

Comment:
Same here, I suppose this step should pass a descriptor with dimensions `[batchSize, hiddenSize]`.

Reply:
Done in 622ce7535c9a4ded50526a12e905b5bf75b36ecc